### PR TITLE
Release notes for Istio 1.14.2

### PR DESCRIPTION
**Pull request by lei-tang**
Please provide a description for what this PR is for: 
Release notes for Istio 1.14.2.

And to help us figure out who should review this PR, please 
put an X in all the areas that this PR affects.

- [ ] Configuration Infrastructure
- [X ] Docs
- [ ] Installation
- [ ] Networking
- [ ] Performance and Scalability
- [ ] Policies and Telemetry
- [ ] Security
- [X ] Test and Release
- [ ] User Experience
- [ ] Developer Infrastructure

### CODE REVIEW

**Comment by GregHanson** [comment]
should include note about go 1.18.4 security patch: https://groups.google.com/g/golang-announce/c/nqrv9fbR0zE
Istio not at risk, just FYI

**Comment by lei-tang** [comment]
> should include note about go 1.18.4 security patch: https://groups.google.com/g/golang-announce/c/nqrv9fbR0zE Istio not at risk, just FYI

@GregHanson Thanks! I have added a FYI note for the Go 1.18.4 security fixes. PTAL.

File: content/en/news/releases/1.14.x/announcing-1.14.2/index.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
+---
+title: Announcing Istio 1.14.2
+linktitle: 1.14.2
+subtitle: Patch Release
+description: Istio 1.14.2 patch release.
+publishdate: 2022-07-25
+release: 1.14.2
+aliases:
+- /news/announcing-1.14.2
```
Comment (ericvn) [inline content/en/news/releases/1.14.x/announcing-1.14.2/index.md:9]: No need for the alias

File: content/en/news/releases/1.14.x/announcing-1.14.2/index.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+This release note describes what’s different between Istio 1.14.1 and Istio 1.14.2.
+
+FYI, [Go 1.18.4 has been released](https://groups.google.com/g/golang-announce/c/nqrv9fbR0zE),
+which includes 9 security fixes. We recommend you to upgrade to this newer Go version.
+
+{{< relnote >}}
+
+# Changes
+
+- **Added** `istioctl experimental envoy-stats -o prom-merged` for retrieving `istio-proxy` merged metrics from prometheus.
```
Comment (ericvn) [inline content/en/news/releases/1.14.x/announcing-1.14.2/index.md:22]: Prometheus

File: content/en/news/releases/1.14.x/announcing-1.14.2/index.md [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+  ([Issue #39454](https://github.com/istio/istio/issues/39454))
+
+- **Added** support for Kubernetes 1.25 by using new `HorizontalPodAutoscaler` and `PodDisruptionBudget` API versions when supported.
+
+- **Added** the ability to read `kubernetes.io/tls` type `cacerts` secrets.
+  ([Issue #38528](https://github.com/istio/istio/issues/38528))
+
+- **Fixed** bug when updating a multi-cluster secret, the previous cluster is not stopped. Even delete the secret can not stop the previous cluster  ([Issue #39366](https://github.com/istio/istio/issues/39366))
```
Comment (ericvn) [inline content/en/news/releases/1.14.x/announcing-1.14.2/index.md:30]: **a** bug
delet**ing**
can/will
Add a period

File: content/en/news/releases/1.14.x/announcing-1.14.2/index.md [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+
+- **Fixed** a bug where specifying `warmupDuration` without `Lb` policy is not configuring warmup duration.  ([Issue #39430](https://github.com/istio/istio/issues/39430))
```
Comment (ericvn) [inline content/en/news/releases/1.14.x/announcing-1.14.2/index.md:32]: **the** warmup

File: content/en/news/releases/1.14.x/announcing-1.14.2/index.md [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+
+- **Fixed** a bug when sending access logging to injected `OTel-collector` pod throws `http2.invalid.header.field` error.  ([Issue #39196](https://github.com/istio/istio/issues/39196))
```
Comment (ericvn) [inline content/en/news/releases/1.14.x/announcing-1.14.2/index.md:34]: throws **a**

File: content/en/news/releases/1.14.x/announcing-1.14.2/index.md [hunk 6]
```diff
@@ -0,0 +1,42 @@
...
+
+- **Fixed** an issue where Istio is sending traffic to unready pods when `PILOT_SEND_UNHEALTHY_ENDPOINTS` is enabled.
+  ([Issue #39825](https://github.com/istio/istio/issues/39825))
+
+- **Fixed** an issue causing Service merging to only take into account the first and last Service, rather than all of them.
+
+- **Fixed** an issue where `ProxyConfig` image type not taking effect
```
Comment (ericvn) [inline content/en/news/releases/1.14.x/announcing-1.14.2/index.md:41]: ... where **the* ... type **is**

**Comment by ericvn** [comment]
You also need to update the data/args.yml file with the newer 1.14.2 version. This is only needed for the 1.14 branch RN PRs. I suspect it will cause an issue with the cherry-pick, but we can fix that later.

File: content/en/news/releases/1.14.x/announcing-1.14.2/index.md (hunk 1 above)
Comment (lei-tang) [reply content/en/news/releases/1.14.x/announcing-1.14.2/index.md]: Removed.

File: content/en/news/releases/1.14.x/announcing-1.14.2/index.md (hunk 2 above)
Comment (lei-tang) [reply content/en/news/releases/1.14.x/announcing-1.14.2/index.md]: Done.

File: content/en/news/releases/1.14.x/announcing-1.14.2/index.md (hunk 3 above)
Comment (lei-tang) [reply content/en/news/releases/1.14.x/announcing-1.14.2/index.md]: Done.

File: content/en/news/releases/1.14.x/announcing-1.14.2/index.md (hunk 4 above)
Comment (lei-tang) [reply content/en/news/releases/1.14.x/announcing-1.14.2/index.md]: Done.

File: content/en/news/releases/1.14.x/announcing-1.14.2/index.md (hunk 5 above)
Comment (lei-tang) [reply content/en/news/releases/1.14.x/announcing-1.14.2/index.md]: Done.

File: content/en/news/releases/1.14.x/announcing-1.14.2/index.md (hunk 6 above)
Comment (lei-tang) [reply content/en/news/releases/1.14.x/announcing-1.14.2/index.md]: Done.

**Comment by lei-tang** [comment]
> You also need to update the data/args.yml file with the newer 1.14.2 version. This is only needed for the 1.14 branch RN PRs. I suspect it will cause an issue with the cherry-pick, but we can fix that later.

Thanks! The data/args.yml file is updated.

File: content/en/news/releases/1.14.x/announcing-1.14.2/index.md [hunk 7]
```diff
@@ -0,0 +1,40 @@
...
+- **Fixed** a bug where specifying `warmupDuration` without `Lb` policy is not configuring the warmup duration.  ([Issue #39430](https://github.com/istio/istio/issues/39430))
+
+- **Fixed** a bug when sending access logging to injected `OTel-collector` pod throws a `http2.invalid.header.field` error.  ([Issue #39196](https://github.com/istio/istio/issues/39196))
+
+- **Fixed** an issue where Istio is sending traffic to unready pods when `PILOT_SEND_UNHEALTHY_ENDPOINTS` is enabled.
+  ([Issue #39825](https://github.com/istio/istio/issues/39825))
+
+- **Fixed** an issue causing Service merging to only take into account the first and last Service, rather than all of them.
+
+- **Fixed** an issue where the `ProxyConfig` image type is not taking effect
```
Comment (ericvn) [inline content/en/news/releases/1.14.x/announcing-1.14.2/index.md:39]: This line needs a period.

Comment (lei-tang) [reply content/en/news/releases/1.14.x/announcing-1.14.2/index.md]: Done.

File: content/en/news/releases/1.14.x/announcing-1.14.2/index.md [hunk 8]
```diff
@@ -0,0 +1,40 @@
...
+description: Istio 1.14.2 patch release.
+publishdate: 2022-07-25
+release: 1.14.2
+---
+
+This release contains bug fixes to improve robustness and some additional support.
+This release note describes what’s different between Istio 1.14.1 and Istio 1.14.2.
+
+FYI, [Go 1.18.4 has been released](https://groups.google.com/g/golang-announce/c/nqrv9fbR0zE),
+which includes 9 security fixes. We recommend you to upgrade to this newer Go version.
```
Comment (ericvn) [inline content/en/news/releases/1.14.x/announcing-1.14.2/index.md:14]: Note we have a comment thread in the 1.13.6 release notes about the Go update as well.

Comment (lei-tang) [reply content/en/news/releases/1.14.x/announcing-1.14.2/index.md]: Changed to "We recommend you to upgrade to this newer Go version if you are using Go locally."

**Review by ericvn — APPROVED** [review]

**Pull request merged**